operations/helm/charts/mimir-distributed/values.yaml

```
@@ -350,6 +350,16 @@ alertmanager:
       cpu: 10m
       memory: 32Mi
 
+  # -- fallback config for alertmanager 
```

Can you also add a sentence which explains when this is used? I know that's described in the mimir/GEM docs, but I think it will be useful to say here too

---

Also, capitalization 
```suggestion
  # -- Fallback config for alertmanager 
```